test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_default_config_test.clj

```diff
@@ -88,19 +88,26 @@ react accordingly."
           "Unexpected default for proxy 'idle-timeout'")
       (.stop client))))
 
-(def selector-thread-count
-  "The number of selector threads that should be allocated per connector.  See:
-   https://github.com/eclipse/jetty.project/blob/jetty-9.4.1.v20170120/jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java#L223
-   and
-   https://github.com/eclipse/jetty.project/blob/jetty-9.4.1.v20170120/jetty-server/src/main/java/org/eclipse/jetty/server/Server.java#L403-L408
-   The number of selectors is twice the number of selector threads."
-  (* 2 (max 1 (min 4 (int (/ (ks/num-cpus) 2))))))
+(defn selector-thread-count
+  [max-threads]
+  "The number of selectors and selecto threads that should be allocated per
+  connector.
+   https://github.com/eclipse/jetty.project/blob/jetty-9.4.11.v20180605/jetty-io/src/main/java/org/eclipse/jetty/io/SelectorManager.java#L70-L74"
+  (max 1
+       (min (int (/ max-threads 16))
+            (int (/ (ks/num-cpus) 2)))))
 
 (def acceptor-thread-count
   "The number of acceptor threads that should be allocated per connector.  See:
-   https://github.com/eclipse/jetty.project/blob/jetty-9.4.1.v20170120/jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnector.java#L194"
+   https://github.com/eclipse/jetty.project/blob/jetty-9.4.11.v20180605/jetty-server/src/main/java/org/eclipse/jetty/server/AbstractConnector.java#L202"
   (max 1 (min 4 (int (/ (ks/num-cpus) 8)))))
 
+(defn reserved-thread-count
```

Comment:
Should I write about ReservedThreadPool here?

Reply:
sure!

Reply:
what do you think of the content now (with a quick blurb and a link)?

Reply:
That works for me, it's in line with the others.